docs/basics/app-anatomy.md

```diff
@@ -158,7 +158,7 @@ Each module should also implement the `RegisterServices` method as part of the [
 
 #### Handlers
 
-The [handler](../building-modules/handler.md) refers to the part of the module responsible for processing the `Msg` after it is routed by `baseapp`. Handler functions of modules are only executed if the transaction is relayed from Tendermint by the `DeliverTx` ABCI message. If the transaction is relayed by `CheckTx`, only stateless checks and fee-related stateful checks are performed. To better understand the difference between `DeliverTx`and `CheckTx`, as well as the difference between stateful and stateless checks, click [here](./tx-lifecycle.md).
+The [handler](../building-modules/msg-services.md#handler-type) refers to the part of the module responsible for processing the `Msg` after it is routed by `baseapp`. Handler functions of modules are only executed if the transaction is relayed from Tendermint by the `DeliverTx` ABCI message. If the transaction is relayed by `CheckTx`, only stateless checks and fee-related stateful checks are performed. To better understand the difference between `DeliverTx`and `CheckTx`, as well as the difference between stateful and stateless checks, click [here](./tx-lifecycle.md).
 
 The `handler` of a module is generally defined in a file called `handler.go` and consists of:
```

Comment:
Is this still true? Do we still even want to use the handler language and have a `handler.go`. I think we can more or less do away with it in new modules. They can just use `RegisterServices` in `module.go` and avoid a handler altogether.

Reply:
Agreed and there are actually other places where the `handler` language is still being used (eg `core/events.md`,  `core/transactions`...), maybe that should be part of another PR though (since it's not in the building modules section)?

Reply:
sounds good, let's just put that in an issue tracker, to not forget that.

Reply:
@clevinson should we add that to https://github.com/cosmos/cosmos-sdk/issues/7661?

Reply:
I removed all mentions of `handler` in "Basics": https://github.com/cosmos/cosmos-sdk/pull/7782. For other sections (which are not started yet), let's switch the vocabulary to "Msg services" as we update each md file.